ZupIT/ritchie-cli
# Adding flags and tests to delete workspace

Signed-off-by: Henrique Moraes <henrique.moraes@zup.com.br>



### Description

Adding flag support for `rit delete workspace` command. Some of the refactoring done as well in this PR:
* Removed `dirManager` as a dependency, it only checked for a file existence in a single line
* Workspace list no longer lists the default workspace. There is no reason to show it if we will never delete it
* Trying to delete the default workspace issues an error

### How to verify it
Run `rit delete workspace --name=<workspace>` and check it has been deleted successfully

### Changelog

Added flag support for rit delete workspace command
